src/include/stir/numerics/sampling_functions.inl

```diff
@@ -54,6 +57,10 @@ void sample_function_on_regular_grid(Array<3,elemT>& out,
     }                             
 }
 
+/* This function samples a function (usually an interpolator) at the locations specified in
```

Comment:
this is actually not a doxygen comment. It should also be in the .h file (follow the scheme used there for the other function)

Reply:
My bad, just copied the format from another file somewhere. Should align with the above function after the next commit.